### PR TITLE
Ensures the hero button icon is centered within the circle

### LINKED ISSUE #346: Hero button icon not centered
As reported by @andrewconnell in #345, the hero button icon is not centered in the circle when using an `a` tag rather than `button`. See the screenshot below.

image

### PR DESCRIPTION
**Pull request by mikewheaton**
Fixes #346

### CODE REVIEW

**Comment by wdo3650** [comment]
Looks good in all browsers except in Edge for me.

File: src/components/Button/Button.scss [hunk 1]
```diff
@@ -131,6 +131,7 @@
 		font-size: $ms-font-size-s;
 		position: relative;
 		top: -8px;
+    text-align: center;
```
Comment (wdo3650) [inline src/components/Button/Button.scss]: Align to the other CSS properties in this class.

**Comment by mikewheaton** [comment]
The icon centers horizontally in Edge, but it sits about a pixel too low in the circle. This is more an issue with the icon font not having centered icons than a bug with this component. Even if we tweaked it for the plus icon, this component should support any icon and we can't get them all aligned.

**Comment by wdo3650** [comment]
Looks good to me. Approved.

**Comment by Jahnp** [comment]
Agreed--looks good.

**Pull request merged**